migration/checkERC20EthToNearTransfersFinalization/index.ts

```diff
@@ -0,0 +1,234 @@
+/**
+ * NEAR-ETH Bridge Transaction Verification Script
+ *
+ * This script checks for ERC20 token locking events on Ethereum and verifies
+ * their finalization status on the NEAR blockchain.
+ */
+
+import { connect, keyStores } from "near-api-js";
+import dotenv from "dotenv";
+import { Header } from "eth-object";
+import { ethers } from "ethers";
+import { Formatter } from "@ethersproject/providers";
+import process from "node:process";
+import { b } from "@zorsh/zorsh";
+import chalk from "chalk"; // For colorized console output
+import { cleanEnv, str } from "envalid"; // For environment validation
+
+// Load environment variables
+dotenv.config();
+
+// Validate environment variables with envalid
+const ENV = cleanEnv(process.env, {
+	ERC20_LOCKER: str({ desc: "Ethereum address of the ERC20 Locker contract" }),
+	NETWORK_ETH: str({ choices: ["mainnet", "goerli", "sepolia"] }),
+	NETWORK_NEAR: str({ choices: ["mainnet", "testnet"] }),
+	BRIDGE_TOKEN_FACTORY_ACCOUNT_ID: str(),
+});
```

Comment:
We should add INFURA_API_KEY here. 

```
/home/olga/Aurora/MainnetMigration/AuroraEthConnectorMainnetMigration/omni-bridge-migration/migration/checkERC20EthToNearTransfersFinalization/node_modules/.pnpm/envalid@8.0.0/node_modules/envalid/src/middleware.ts:49
          throw new ReferenceError(
                ^


ReferenceError: [envalid] Env var INFURA_API_KEY was accessed but not validated. This var is set in the environment; please add an envalid validator for it.
```

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/290/commits/44613661451caf79f9f7fb138d639fb57734b6f7